examples/js/nodes/materials/PhongNode.js

```diff
@@ -131,7 +131,7 @@ THREE.PhongNode.prototype.build = function( builder ) {
 		var normal = this.normal ? this.normal.buildCode( builder, 'v3' ) : undefined;
 		var normalScale = this.normalScale && this.normal ? this.normalScale.buildCode( builder, 'fv1' ) : undefined;
```

Comment:
Hi @sunag. Great work, BTW!

Sorry, I do not understand your reply... Maybe I am missing something, but what I was pointing out is `bumpScale` is univariate, but `normalScale` is bivariate.